hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionStateNode.java

```diff
@@ -161,7 +161,7 @@ public boolean isStuck() {
     return isInState(State.FAILED_OPEN) && getProcedure() != null;
   }
 
-  public boolean isInTransition() {
+  public boolean isOngoingTRSP() {
```

Comment:
How about `isInTransit`? 

And javadoc that "transit" means something different than "in transition".